GVRf/Framework/backend_oculus/src/main/jni/ovr_activity.cpp

```diff
@@ -112,19 +111,19 @@ namespace gvr {
                                                          mResolveDepthConfiguration, mDepthTextureFormatConfiguration);
     }
 
-RenderTextureInfo  GVRActivity::getRenderTextureInfo(int eye, int index){
+    dummy::RenderTextureInfo*  GVRActivity::getRenderTextureInfo(int eye, int index){
     // for multiview, eye index would be 2
     eye = eye % 2;
     FrameBufferObject fbo = frameBuffer_[eye];
 
-    RenderTextureInfo renderTextureInfo;
-    renderTextureInfo.fboId = fbo.getRenderBufferFBOId(index);
-    renderTextureInfo.fboHeight = fbo.getHeight();
-    renderTextureInfo.fdboWidth = fbo.getWidth();
-    renderTextureInfo.multisamples = mMultisamplesConfiguration;
-    renderTextureInfo.useMultiview = use_multiview;
-    renderTextureInfo.views = use_multiview ? 2 : 1;
-    renderTextureInfo.texId = fbo.getColorTexId(index);
+    dummy::RenderTextureInfo* renderTextureInfo = new dummy::RenderTextureInfo();
```

Comment:
In this particular backend, there is already dependency to the framework. Why not just use directly gvr::RenderTextureInfo?